README.md

```diff
@@ -46,14 +46,16 @@ The scripts in this package leverage the configuration files saved in the [`conf
 
 The library comes with a default configuration set in [`conf/config.yaml`](conf/config.yaml).
 
+> NOTE: Be sure to add your actions to the `augmented_cols` instead of `inputs` in the data config file located at `conf/data`.
```

Comment:
Is there a reason why we want to make this requirement? Either shoudl work. Initiailly, the design was `augmented_cols` would be for exogeneous variables, like configs only.

Reply:
Either should work? Then I can remove this. This was one of the things Sandip suggested for when I finally got things to work with ddm_predictor.py